ZenPacks/zenoss/LinuxMonitor/zenpack.yaml

```diff
@@ -200,6 +250,73 @@ device_classes: !ZenPackSpec
                                 dpName: volume_free
                                 lineType: AREA
                                 rpn: "size,-,size,/,-100,*"
+                    Operation Throughput:
+                        units: operations/sec
+                        miny: 0
+                        maxy: 100
+                        graphpoints:
+                            Reads:
+                                dpName: diskstats_readsCompleted
+                                lineType: LINE
+                            Writes:
+                                dpName: diskstats_writesCompleted
+                                lineType: LINE
+                    Merge Rate:
+                        units: operations merged/sec
+                        miny: 0
+                        maxy: 100
+                        graphpoints:
+                            Reads:
+                                dpName: diskstats_readsMerged
+                                lineType: LINE
+                            Writes:
+                                dpName: diskstats_writesMerged
+                                lineType: LINE
+                    Sector Throughput:
+                        units: sectors/sec
+                        miny: 0
+                        maxy: 100
+                        graphpoints:
+                            Read:
+                                dpName: diskstats_sectorsRead
+                                lineType: LINE
+                            Written:
+                                dpName: diskstats_sectorsWritten
+                                lineType: LINE
+                    IO Throughput:
+                        units: io in progress
+                        miny: 0
+                        maxy: 1000
+                        graphpoints:
+                            IO:
+                                dpName: diskstats_ioInProgress
+                                lineType: LINE
+                    IO Utilization:
+                        units: percent
+                        miny: 0
+                        maxy: 100
+                        graphpoints:
+                            Reading:
+                                dpName: diskstats_msReading
+                                lineType: LINE
+                                rpn: "10,/"
+                            Writing:
+                                dpName: diskstats_msWriting
+                                lineType: LINE
+                                rpn: "10,/"
+                            Doing IO:
+                                dpName: diskstats_msDoingIO
+                                lineType: LINE
+                                rpn: "10,/"
+                    Weighted IO Utilization:
+                        units: percent
```

Comment:
percent tends to indicate 0-100. Maybe be more specific with the units here with something like "weighted percent".